temporal-sdk/src/main/java/io/temporal/worker/Worker.java

```diff
@@ -386,11 +400,28 @@ public void registerActivitiesImplementations(Object... activityImplementations)
     workflowWorker.registerLocalActivityImplementations(activityImplementations);
   }
 
+  /**
+   * Register Nexus service implementation objects with a worker.
+   *
+   * <p>An Nexus service object must implement at least one interface annotated with {@link
+   * io.nexusrpc.handler.ServiceImpl}.
```

Comment:
```suggestion
   * <p>An Nexus service object must be annotated with {@link
   * io.nexusrpc.handler.ServiceImpl}.
```

Technically unrelated to interfaces